### ISSUE TITLE
Bug: gps location is only updating every 60 seconds

### ISSUE THREAD

**Issue by SLMapper** [Helium314/SCEE#533]
In SCEE current position is only updating every minute (I measured the time).
After reopening the app few instant location updates happen, but then updating switches to minutely updates. This makes it very hard to use the app.

Things I tried:
- checking other apps in parallel: they display immediate location updates (OsmAnd, Streetcomplete)
- disabling the previously enabled _quests > notify about nearby quests_ or changing settings for it: no changes
- trying to change "gps location update interval" from its unchanged default 0 seconds to 1 and back: no changes

**How to Reproduce**
(once enable expert mode - not sure if this is required for reproducing)
(once enable _quests > notify about nearby quests_ - not sure if this is required for reproducing)
1. enable gps on your device
2. open the app and walk around

Here is the exported settings.xml (personal information reducted)
[scee-settings-reducted.txt](https://github.com/Helium314/SCEE/files/14904691/scee-settings-reducted.txt)

**Expected Behavior**
location updates are immediately visible like in other apps

**Does it happen in normal StreetComplete?**
no

**Versions affected**
v57.0

**Comment by Helium314** [comment]
duplicate of #524
